core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3459,6 +3459,12 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_FILE_SYSTEM_MERGE_INODE_JOURNALS =
```

Comment:
Having a switch for an experimental feature is reasonable. However, if we assume the implementation here is all correct, will there still be use cases where we want a `false` here? I guess the answer is no? That leads me to think when we can get rid of this property.

Reply:
yeah let's release this to the community and find someone dog fooding this first.

Reply:
Maybe should at least set the default to true?